submodules/eps/EPS.py

```diff
@@ -0,0 +1,187 @@
+import logging
+import time
+import smbus
+import yaml
+
+from smbus2 import SMBusWrapper
+from submodules import telemetry
+from core.log import Log
+from core.error import Error
+# Threadhandler dependencies
+# from functools import partial
+# from core.threadhandler import ThreadHandler
+
+class EPS():
+    def __init__(self, config: dict):
+        self.config = config
+        self.modules = dict()
+        # TODO: Add ThreadHandler object
```

Comment:
if you do end up adding a `Thread` object initialize a dictionary called `self.processes` with keys being strings and the values being `Thread` objects
e.g.
```
self.processes = {
   "eps-listen": ThreadHandler(target=partial(listen), name="eps-listen", ...)
}
```